src/pages/groupComparison/ClinicalDataUtils.ts

```diff
@@ -0,0 +1,7 @@
+import { getServerConfig } from 'config/config';
+
+export function getComparisonCategoricalNaValue(): string[] {
+    const rawString = getServerConfig().comparison_categorical_na_value;
```

Comment:
is it ever possible this value is null or is the default in the server configuration empty string?